config.json

```diff
@@ -1,13 +1,14 @@
 {
-    "IP": "",
-    "EventNotificationChannelIdForShiny": "",
-    "EventNotificationChannelIdForBrilliant": "",
+    "DiscordToken": "ODgzNzg4MTc1MjgwMDc4OTAw.YTPB4A.VlygRJeEkzCV8mUpBt6U8EiNAzg",
+    "EnableDebugLogging": "False",
+    "EventNotificationChannelIdForShiny": "884610291487342622",
+    "EventNotificationChannelIdForBrilliant": "884610733931909240",
     "EventNotificationChannelIdForMark": "",
-    "EventNotificationChannelIdForRareMark": "",
-    "NotificationChannelForInfo": "",
-    "TextNotificationChannelIdForShiny": "",
-    "TextNotificationChannelIdForBrilliant": "",
+    "EventNotificationChannelIdForRareMark": "884610365089017886",
+    "IP": "192.168.86.250",
+    "NotificationChannelForInfo": "883789388289568818",
+    "TextNotificationChannelIdForShiny": "883789388289568818",
+    "TextNotificationChannelIdForBrilliant": "883789388289568818",
     "TextNotificationChannelIdForMark": "",
-    "TextNotificationChannelIdForRareMark": "",
-    "EnableDebugLogging": "False"
+    "TextNotificationChannelIdForRareMark": "883789388289568818"
```

Comment:
remember to set config back to default, also should regenerate your bots token so no one can use it

Reply:
Yeeep, discord caught it and disabled it. I went ahead and did the template thing and excluded config.json using gitignore to prevent this from happening again.